### PR TITLE
Add custom PHP CS Fixer rule to clean migration

### PR DESCRIPTION
**Pull request by JeremieSamson**
Do the exact same thing as #3448 but with php_cs_fixer :) 

No need to update makefile, it will be apply with your `make phpcsfix`

### CODE REVIEW

File: src/Fixer/DoctrineMigrationFixer.php [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+namespace AppBundle\Fixer;
+
+use PhpCsFixer\Fixer\DefinedFixerInterface;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\FixerDefinition\FixerDefinitionInterface;
+use PhpCsFixer\Tokenizer\Tokens;
+
+final class DoctrineMigrationFixer implements DefinedFixerInterface
```
Comment (ottaviano) [inline src/Fixer/DoctrineMigrationFixer.php]: what about extends the `AbstractFixer`?

File: Makefile [hunk 2]
```diff
@@ -77,7 +77,7 @@ db: vendor wait-for-db
 
 db-diff: vendor wait-for-db                                                                            ## Generate a migration by comparing your current database to your mapping information
 	$(CONSOLE) doctrine:migration:diff --formatted
-
+
```
Comment (illusionOfParadise) [inline Makefile]: It's useless.

File: src/Fixer/DoctrineMigrationFixer.php [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+{
+    public function getName(): string
+    {
+        return 'PhpCsFixerCustomFixer/doctrine_migration_clean';
+    }
+
+    public function getDefinition(): FixerDefinitionInterface
+    {
+        return new FixerDefinition(
+            'Migrations files should not have declare(strict_types=1), auto-generated comments, and abortif',
```
Comment (thomasbisignani) [inline src/Fixer/DoctrineMigrationFixer.php]: Not a big deal, but what do you think about : 
`..., auto-generated comments, and abortIf() calls',`

File: src/Fixer/DoctrineMigrationFixer.php [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\FixerDefinition\FixerDefinitionInterface;
+use PhpCsFixer\Tokenizer\Tokens;
+
+final class DoctrineMigrationFixer implements DefinedFixerInterface
+{
+    public function getName(): string
+    {
+        return 'PhpCsFixerCustomFixer/doctrine_migration_clean';
```
[the remark this answers is not in the document]
Comment (ottaviano) [reply src/Fixer/DoctrineMigrationFixer.php]: @JeremieSamson, can we remove the prefix `PhpCsFixerCustomFixer`?

**Comment by thomasbisignani** [comment]
:ok_hand: :medal_sports:

**Pull request merged**